code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -84,6 +84,28 @@
 		if(standard_pour_into(user, target))
 			return
 
+		//Help the user avoid clumsy splashing mistakes due to finicky sprites etc.
+		//for lidded containers as well as effects (radium puddles, blood, etc.)
+		if  (user.clumsy == 0 && (istype(target, /obj/item/weapon/reagent_containers)))
+			var/resp = alert(user, "Do you really want to splash the solution?", "Splash the solution?", "Abort", "Proceed")
+			if (resp == "Abort")
+				return
+			else if (get_dist(user, target)>1)
```

Comment:
This will ignore, among other things: you having lost the beaker by this point, there being a glass between you and the target, and you being dead. I'm not sure how but this needs better handling.